### PR TITLE
wireframes, actions and issues/pr templates

### PR DESCRIPTION
**Pull request by ecureuill**
## Types of changes
What types of changes does your code introduce?

 

 

- [ ] Bugfix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected) 
- [x] Documentation Update (if none of the other choices apply)

## What is the current behavior? 

Update issues and pr templates
Update actions
Add wire-frames for feature specification

### CODE REVIEW

**Comment by ecureuill** [comment]
PR for test.
@rsg73626 please review the PR adding code suggestion like I did

Action didn't triggered by new PR :trollface:

**Comment by rsg73626** [comment]
Testing. General comment in the PR as a whole.

File: .github/.gitignore [hunk 1]
```diff
@@ -0,0 +1 @@
+*.yml
```
Comment (rsg73626) [inline .github/.gitignore:1]: General comment in a changed line.

Comment (rsg73626) [reply .github/.gitignore:1]: Starting a review in this specific line (I do not know what it means exactly).

Comment (rsg73626) [inline .github/.gitignore:1]:
```suggestion
suggetion (test)
```

Comment (ecureuill) [reply .github/.gitignore:1]: bla

File: .github/workflows/main.yml [hunk 2]
```diff
@@ -110,20 +110,29 @@ jobs:
         if: |
           github.event_name == 'issues' && 
           github.event.action == 'labeled' &&  
-          contains(github.event.issue.labels.*.name, ':bulb:Feature Request')
+          contains(github.event.issue.labels.*.name, 'Feature Request')
```
Comment (ecureuill) [inline .github/workflows/main.yml:113]:
```suggestion
          contains(github.event.issue.labels.*.name, ':bulb:Feature Request')
```

File: .github/workflows/main.yml [hunk 3]
```diff
@@ -110,20 +110,29 @@ jobs:
...
         with:
-          project: 'https://github.com/solex2006/SELIProject/projects/6'
+          project: 'https://github.com/ecureuill/SELIProject/projects/3'
```
Comment (ecureuill) [inline .github/workflows/main.yml:115]:
```suggestion
          project: 'https://github.com/solex2006/SELIProject/projects/6'
```

File: .github/workflows/main.yml [hunk 4]
```diff
@@ -110,20 +110,29 @@ jobs:
...
           column_name: 'NEEDS TRIAGE'
+<<<<<<< HEAD
```
Comment (ecureuill) [inline .github/workflows/main.yml:117]:
```suggestion

```

File: .github/workflows/main.yml [hunk 5]
```diff
@@ -110,20 +110,29 @@ jobs:
...
+        if: |
+          github.event_name == 'issues' && 
+          github.event.action == 'labeled' &&
+          contains(github.event.issue.labels.*.name, 'bug')
+=======
       - name: Assign 'BUG', 'ISSUE',' BUG ON NEXT RELEASE' label to 'TRIAGE - Bugs'
         uses: srggrs/assign-one-project-github-action@1.2.0
         if: |
           github.event_name == 'issues' && 
           github.event.action == 'labeled' &&  (
```
Comment (ecureuill) [inline .github/workflows/main.yml:129]:
```suggestion
=======

```

File: .github/workflows/main.yml [hunk 6]
```diff
@@ -110,20 +110,29 @@ jobs:
...
-          contains(github.event.issue.labels.*.name, ':bug:bug') ||
-          contains(github.event.issue.labels.*.name, ':beetle:bug-next') ||
-          contains(github.event.issue.labels.*.name, ':ant:issue'))
+          contains(github.event.issue.labels.*.name, 'bug') ||
+          contains(github.event.issue.labels.*.name, 'bug on next release') ||
+          contains(github.event.issue.labels.*.name, 'issues'))
+>>>>>>> 8e62ccf86425a1d6ee9ce8920222483800ab848e
```
Comment (ecureuill) [inline .github/workflows/main.yml:133]:
```suggestion

```

File: .github/workflows/main.yml [hunk 7]
```diff
@@ -110,20 +110,29 @@ jobs:
...
         with:
-          project: 'https://github.com/solex2006/SELIProject/projects/7'
+          project: 'https://github.com/ecureuill/SELIProject/projects/1'
```
Comment (ecureuill) [inline .github/workflows/main.yml:135]:
```suggestion
          project: 'https://github.com/solex2006/SELIProject/projects/7'
```

File: .github/workflows/main.yml [hunk 8]
```diff
@@ -134,7 +143,7 @@ jobs:
           contains(github.event.issue.labels.*.name, '1 - Planning') ||
           contains(github.event.issue.labels.*.name, 'need specification'))
         with:
-          project: 'https://github.com/solex2006/SELIProject/projects/8'
+          project: 'https://github.com/ecureuill/SELIProject/projects/4'
```
Comment (ecureuill) [inline .github/workflows/main.yml:146]:
```suggestion
          project: 'https://github.com/solex2006/SELIProject/projects/8'
```

File: .github/workflows/main.yml [hunk 9]
```diff
@@ -144,7 +153,7 @@ jobs:
           github.event.action == 'labeled' &&  
           contains(github.event.issue.labels.*.name, '2 - Ready')
         with:
-          project: 'https://github.com/solex2006/SELIProject/projects/9'
+          project: 'https://github.com/ecureuill/SELIProject/projects/2'
```
Comment (ecureuill) [inline .github/workflows/main.yml:156]:
```suggestion
          project: 'https://github.com/solex2006/SELIProject/projects/9'
```

File: .github/workflows/main.yml [hunk 10]
```diff
@@ -154,7 +163,7 @@ jobs:
           github.event.action == 'labeled' &&  
           contains(github.event.issue.labels.*.name, 'need tester')
         with:
-          project: 'https://github.com/solex2006/SELIProject/projects/10'
+          project: 'https://github.com/ecureuill/SELIProject/projects/6'
```
Comment (ecureuill) [inline .github/workflows/main.yml:166]:
```suggestion
          project: 'https://github.com/solex2006/SELIProject/projects/10'
```

File: .github/workflows/main.yml [hunk 11]
```diff
@@ -163,5 +172,5 @@ jobs:
           github.event_name == 'pull_request' &&
           github.event.action == 'opened'
         with:
-          project: 'https://github.com/solex2006/SELIProject/projects/11'
-          column_name: 'Review in progress'
+          project: 'https://github.com/ecureuill/SELIProject/projects/7'
+          column_name: 'In progress'
```
Comment (ecureuill) [inline .github/workflows/main.yml:176]:
```suggestion
          project: 'https://github.com/solex2006/SELIProject/projects/11'
          column_name: 'Review in progress'
```

**Pull request merged**